.github/workflows/build.yml

```diff
@@ -72,15 +72,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        kubernetes: [v1.20.6]
+        kubernetes: [v1.19.16, v1.25.0]
```

Comment:
To check if the build passed for kubernetes v1.19 and v1.25. It passed maybe because the volumes were not HA, and PDB object was not created. v1.19 should be removed. Good catch!